linkis-computation-governance/linkis-manager/linkis-resource-manager/src/main/java/org/apache/linkis/resourcemanager/utils/RequestKerberosUrlUtils.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright 2019 WeBank
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
```

Comment:
The license needs to be modified to apache format

Reply:
Hi Wang!
 I have modified it.